CDP4Composition/ViewModels/PluginRowViewModel.cs

```diff
@@ -0,0 +1,254 @@
+        /// Gets or Sets the <see cref="author"/> of the represented plugin
+        /// </summary>
+        public string Author
+        {
+            get => this.author;
+            set => this.RaiseAndSetIfChanged(ref this.author, value);
+        }
+
+        /// <summary>
+        /// Backing field for the property <see cref="ReleaseNote"/>
+        /// </summary>
+        private string releaseNote;
+
+        /// <summary>
+        /// Gets or Sets the <see cref="author"/> of the represented plugin
+        /// </summary>
+        public string ReleaseNote
+        {
+            get => this.releaseNote;
+            set => this.RaiseAndSetIfChanged(ref this.releaseNote, value);
+        }
+
+        /// <summary>
+        /// Backing field for the property <see cref="InstallationProgress"/>
+        /// </summary>
+        private double installationProgress;
+
+        /// <summary>
+        /// Gets or Sets the <see cref="installationProgress"/> of the represented plugin
+        /// </summary>
+        public double InstallationProgress
+        {
+            get => this.installationProgress;
+            set => this.RaiseAndSetIfChanged(ref this.installationProgress, value);
+        }
+
+        /// <summary>
+        /// Backing field for the property <see cref="IsSelectedForInstallation"/>
+        /// </summary>
+        private bool isSelectedForInstallation;
+        
+        /// <summary>
+        /// Gets or sets the assert <see cref="isSelectedForInstallation"/> whether the represented plugin will be installed
+        /// </summary>
+        public bool IsSelectedForInstallation
+        {
+            get => this.isSelectedForInstallation;
+            set => this.RaiseAndSetIfChanged(ref this.isSelectedForInstallation, value);
+        }
+        
+        /// <summary>
+        /// Gets the <see cref="IPluginFileSystemService"/> to operate on
+        /// </summary>
+        public IPluginFileSystemService FileSystem { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Command that open the plugin store web page relative to this represented plugin
+        /// </summary>
+        public ReactiveCommand<object> WebsiteCommand { get; private set; }
+        
+        /// <summary>
+        /// Gets the plugin download path and the new manifest
+        /// </summary>
+        public (FileInfo cdp4ckFile, Manifest manifest) Plugin { get; private set; }
+
+        /// <summary>
+        /// Instanciate a new <see cref="PluginRowViewModel"/>
+        /// </summary>
+        /// <param name="plugin">The represented plugin</param>
+        /// <param name="pluginFileSystemService">The file system to operate on</param>
+        public PluginRowViewModel((FileInfo cdp4ckFile, Manifest manifest) plugin, IPluginFileSystemService pluginFileSystemService = null)
+        {
+            this.Plugin = plugin;
+            this.FileSystem = pluginFileSystemService ?? new PluginFileSystemService(plugin);
```

Comment:
please make use of MEF to do DI

Reply:
I wish I could but the container is not initialized at this point